src/frontends/pytorch/src/op/norm.cpp

```diff
@@ -326,6 +327,63 @@ OutputVector translate_frobenius_norm(const NodeContext& context) {
     return {result};
 }
 
+OutputVector translate_rms_norm(const NodeContext& context) {
```

Comment:
btw, I think it is better to create separate file `rms_norm`. It will be easy to find it in sources